rest-service/manager_rest/test/base_test.py

```diff
@@ -59,6 +60,11 @@
 )
 
 
+MIGRATION_DIR = os.path.normpath(os.path.join(
```

Comment:
this is unfortunate, but we must use migrations to set up the db in unittests, not create_all